Resources/config/consumer.xml

```diff
@@ -17,6 +17,7 @@
             <argument>sonata.media.thumbnail.format</argument>
             <argument type="service" id="sonata.media.thumbnail.format"/>
             <argument type="service" id="sonata.notification.backend" />
+            <argument type="service" id="event_dispatcher" />
```

Comment:
Not necessary. You can replace it by `<argument />` with noting inside and use a `CompilerPass` to add it.

But I prefer wait for @rande confirmation.